app/components/OceanInterface/OceanInterface.tsx

```diff
@@ -11,71 +11,98 @@ import { useWhaleApiClient } from '../../contexts/WhaleContext'
+//   const supported = await Linking.canOpenURL(url)
+//   if (supported) {
+//     await Linking.openURL(url)
+//   }
+// }
+
+function copyToClipboard (txid: string): void {
+  const msg = `Transaction broadcasted, block should be mined and confirmed wihin next minute, txid: ${txid}`
+  Clipboard.setString(msg)
 }
 
-async function broadcastTransaction (tx: OceanTransaction, client: WhaleApiClient, setError: (e: Error) => void): Promise<string | undefined> {
-  let retries = 0
+async function broadcastTransaction (tx: OceanTransaction, client: WhaleApiClient, retries: number = 0): Promise<string> {
   try {
     return await client.transactions.send({ hex: tx.signed.toHex() })
   } catch (e) {
-    retries++
     Logging.error(e)
-    if (retries < 1) {
-      return await broadcastTransaction(tx, client, setError)
+    if (retries < MAX_AUTO_RETRY) {
+      return await broadcastTransaction(tx, client, retries + 1)
     }
-    setError(e)
+    throw e
   }
 }
 
 /**
  * @description - Global component to be used for async calls, network errors etc. This component is positioned above the bottom tab.
  * Need to get the height of bottom tab via `useBottomTabBarHeight()` hook to be called on screen.
  * */
-export function OceanInterface (): JSX.Element {
-  const { height, err: e } = useSelector((state: RootState) => state.ocean)
-  const transaction = useSelector((state: RootState) => firstTransactionSelector(state.ocean))
-  const [tx, setTx] = useState<OceanTransaction>(transaction)
+export function OceanInterface (): JSX.Element | null {
   const dispatch = useDispatch()
   const client = useWhaleApiClient()
+
+  // store
+  const { height, err: e } = useSelector((state: RootState) => state.ocean)
+  const transaction = useSelector((state: RootState) => firstTransactionSelector(state.ocean))
+
+  // state
+  const [tx, setTx] = useState<OceanTransaction|undefined>(transaction)
   const [err, setError] = useState<Error | undefined>(e)
+
   const slideAnim = useRef(new Animated.Value(0)).current
   Animated.timing(slideAnim, { toValue: height, duration: 300, useNativeDriver: false }).start()
 
+  const dismissDrawer = useCallback(() => {
+    setTx(undefined)
+    setError(undefined)
+  }, [])
+
   useEffect(() => {
+    // last available job will remained in this UI state until get dismissed
     if (transaction !== undefined) {
-      broadcastTransaction(transaction, client, setError).then(() => {
-        setTx({
-          ...transaction,
-          title: translate('screens/OceanInterface', 'Transaction complete'),
-          broadcasted: true
-        })
-        dispatch(ocean.actions.popTransaction())
-      }).catch(() => {
+      console.log('broadcasting', transaction.signed.txId)
+      setTx({
+        ...transaction,
+        broadcasted: false
       })
+
+      broadcastTransaction(transaction, client)
+        .then(() => setTx({ ...transaction, broadcasted: true, title: translate('screens/OceanInterface', 'Sent') }))
```

Comment:
Update the local UI state to display success tx broadcast. UI will be replaced by next transaction if any in the queue.

Reply:
Let's keep the copy as "Transaction complete"/"Transaction sent". "Sent" can mean many things and quite be confusing for different operations (DEX, LM etc.)